zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/ZakenApiPlugin.kt

```diff
@@ -220,6 +223,57 @@ class ZakenApiPlugin(
         )
     }
 
+    @PluginAction(
+        key = "set-zaakstatus",
+        title = "Set zaak status",
+        description = "Sets the status of a zaak",
+        activityTypes = [ActivityType.SERVICE_TASK_START]
+    )
+    fun setZaakStatus(
+        execution: DelegateExecution,
+        @PluginActionProperty statustypeUrl: URI,
+        @PluginActionProperty statustoelichting: String?,
+    ) {
+        val documentId = UUID.fromString(execution.businessKey)
+        val zaakUrl = zaakUrlProvider.getZaakUrl(documentId)
```

Comment:
It could fail on this, right? If there's no zaak that can be found. Should we throw a more specific exception here? Same goes for `createZaakResultaat` as it is using the same method.

Reply:
The function `getZaakUrl(..)` will throw an `ZaakInstanceLinkNotFoundException` when there is no zaak found for that document